### ISSUE TITLE
[BUG] agbsave options missing :(

### ISSUE THREAD

**Issue by JakeYankovich** [d0k3/GodMode9#831]
Full story (long version):
I am attempting to use agbsave options to dump a save from my pokemon crystal (GBC) virtual console.
I took a pokemon crystal GBC rom and used New Super Ultimate Injector (NSUI) to create a CIA file. I installed it as usual and eventually realized 4 of my pokemon require trade for evolution 🙄. I attempted to trade with my other 3ds with the same game and it would not connect.
I decided to instead attempt to dump my save file so I could use some other program to cheat and evolve my team. I found these instructions to dump my save file using godmode9: https://digiex.net/threads/3ds-gba-save-backup-and-restore-with-godmode9.15395/
Unfortunately when I navigate to agbsave.bin I do not see `AGBSAVE options...`.

**TLDR:**
- Attempting to dump save using godmode9 according to [this guide](https://digiex.net/threads/3ds-gba-save-backup-and-restore-with-godmode9.15395/)
- `AGBSAVE options...` is not available

**Info about my system**
Include this info to make our work easier:
* original (not "new") 3ds XL
* GodMode9 v2.1.1
* yes I am using boot9strap and luma

I am attaching a video showing what I am trying to do, as well as my agbsave.bin

**Have you actually read this?**
[X] I have read the information above

**Comment by JakeYankovich** [comment]
[google drive link with video and agbsave.bin](https://drive.google.com/drive/folders/14z6n4lBbna8ck0aGFZqdOiHKm39kHH84?usp=drive_link)

**Comment by GerbilSoft** [comment]
GBC Virtual Console titles are full emulators; they don't use AGB_FIRM, and hence don't use agbsave.bin. You'd need to dump the save file from the GBC Virtual Console title.

**Comment by JakeYankovich** [comment]
This is helpful.
It sounds like perhaps everything is behaving correctly ?
I still am unsure how to dump my save file

**Comment by JakeYankovich** [comment]
Okay awesome, I learned I can just go in "Checkpoint" and make a save (sav.dat) and then just pull it off my sd card and rename it (something.sav). Then I just had to find the tool that converts between .sav and .esv for goomba...
